src/SQL/Builder/WithSQLBuilder.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\SQL\Builder;
+
+use Doctrine\DBAL\Query\WithQuery;
+
+interface WithSQLBuilder
```

Comment:
Does this need to be an interface? Could we make it a class?

Reply:
Right, for now we do not need other implementation of this interface